modules/apps/dynamic-data-mapping/dynamic-data-mapping-lang/src/main/resources/content/Language.properties

```diff
@@ -127,6 +127,7 @@ edit-element-set=Edit Element Set
 edit-x-field=Edit {0}
 element-sets=Element Sets
 empty-list=No results were found.
+enable-alphabetical-order=Enable alphabetical order
```

Comment:
Hi @robertoDiaz,

Could you please change the sentence to `Order Options Alphabetically`? Thanks!